src/test/java/io/projectriff/invoker/FunctionalTests.java

```diff
@@ -67,7 +67,9 @@ public void check() {
 
 	@Test
 	public void fatJar() throws Exception {
-		runner.run("--server.port=" + port, "--function.uri=" + sampleJar.toURI());
+		runner.run("--server.port=" + port, "--function.uri=" + sampleJar.toURI(),
+				"--spring.functional.enabled=true");
+		Thread.sleep(1000L); // Sometimes netty hasn't quite started yet
```

Comment:
that line is a bit scary to me :)
is there an issue somewhere that tracks the problem?

Reply:
No. Feel free to open one.